analysis/src/vm/batcher.rs

```diff
@@ -162,7 +162,9 @@ mod tests {
             env!("CARGO_MANIFEST_DIR")
         ));
         let file_name = base_path.join(path);
-        let expected = fs::read_to_string(file_name).unwrap();
+        let expected = fs::read_to_string(file_name)
+            .unwrap()
+            .replace("machine Main", "machine ::Main");
```

Comment:
Shouldn't this be present in the test already?

Reply:
Yeah, this might be a bug, I'm investigating currently.

Reply:
Ah right, it is fixed, it's not part of the diff anymore.